evildmp/vanilla-sphinx-test
# Add custom CSS for notes and admonitions

## Done

- Add custom CSS for the admonition pattern. We can't use Vanilla directly as the markup on Sphinx is slightly different. This is a kind of work around

## QA

- Check out this branch
- run `make run`
- Go to `/about-repository/repertoire/#sub-sub-sub-heading`
- Check the banners look the same as https://vanillaframework.io/docs/patterns/notification
